tests/test_general_api.py

```diff
@@ -64,6 +64,8 @@ def test_multifield_classify(self):
         Ensure saving + loading does not cause errors
         Ensure saving + loading does not change predictions
         """
+        self.model.hparams.batch_size = 2
```

Comment:
We can create a test hparams and pass it at construction time. 

```
def get_test_hparams():
    hparams = get_default_hparams()
    hparams.hvu9rhu90e = hvufe
    ...
    return hparams
```
I think this is a little tidier? I think changing hparams after model construction is reasonably undefined behaviour.